components/cards/cards.css

```diff
@@ -0,0 +1,147 @@
+@import url(/components/colors.css);
+.card {
+  width: 20rem;
+  border: 2px solid var(--light-secondary-color);
+  border-radius: 0.5rem;
+  height: fit-content;
+  position: relative;
+}
+.text-card {
+  padding: 1rem;
+}
+
+.card-content {
+  margin-top: 0.75rem;
+}
+.card:hover {
+  border: 2px solid var(--secondary-color);
+}
+
+.responsive-img {
+  width: 100%; /* TODO:move this to img */
+}
+.rounded-top-corner-img {
+  border-radius: 0.3rem;
+  border-bottom-right-radius: 0;
+  border-bottom-left-radius: 0;
+}
+/*** vertical-card ***/
+.card-heading {
+  font-weight: 400;
+  padding: 0.5rem 0.75rem;
+}
+.card-sub-heading {
+  font-weight: 500;
+  padding: 0 0.75rem;
+}
+.card-paragrph {
+  font-size: 0.875rem;
+  font-weight: 400;
+  margin-bottom: 1rem;
+  padding: 0 0.75rem;
+}
+.card-call-to-action {
+  display: flex;
+  column-gap: 1rem;
+  row-gap: 1rem;
+  flex-wrap: wrap;
+  margin: 0.5rem;
+}
+/*******/
+
+/*** horizontal-card ***/
+.horizontal-card {
+  display: flex;
+  column-gap: 1rem;
+  width: 30rem;
+}
+.horizontal-card .card-image-container {
+  width: 30%;
+  flex-shrink: 0;
+}
+.horizontal-card .card-image-container img {
+  height: 100%;
+}
+/*******/
+
+/*** card-with-dismiss ***/
+.card-close-btn {
+  background-color: transparent;
+  border: none;
+  font-size: 1.2rem;
+  position: absolute;
+  top: 5px;
+  right: 5px;
+  opacity: 0.5;
+}
+.card-close-btn:hover {
+  opacity: 1;
+}
+/*******/
+
+/**** card-with-badge ***/
+.card-badge {
+  background-color: var(--primary-color);
+  color: var(--secondary-color);
+  padding: 1px 6px;
+  border-radius: 0.3rem;
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+/*******/
+
+/*** cards-with-text-overlay ***/
+.overlay-container {
+  position: absolute;
+  inset: 0 0 0 0;
+  background-color: rgb(234, 234, 236, 0.6);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.overlay-text {
+  font-size: 2.5rem;
+  background-color: #eaeaec;
+  font-weight: 500;
+}
+/*******/
+
+/*** shadow ***/
+.shadow {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
```

Comment:
make variable for shadows also

Reply:
On it. Thanks for the review!

> Hello mahesh. Amazing cards. I just want to add that.. ![image](https://user-images.githubusercontent.com/58331578/152237872-54a1189f-b2f8-4645-933e-f439e8c0b8f0.png) This card seems like too big in height. You can reduce the height of card, it will look good. Otherwise rest of things are good.

Hey vijay, I just realized that I was using the wrong sized image for the card. That image was not meant to be used there fixing the issue now. Thanks for the review.

Reply:
On it. Thanks for the review.